src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1350,6 +1350,9 @@
    "CREATE INDEX idx_reports_producer_timestamp
       ON reports(producer_timestamp)"))
 
+(defn add-code-id-to-catalogs []
+  (jdbc/do-commands "ALTER TABLE catalogs ADD code_id VARCHAR(255)"))
```

Comment:
as I understand it with postgres there's no upside to varchar over text here if we have no reason to cap the length. They'll be stored the same way, except varchar will involve some minimal-overhead bounds checking we don't need. text saves us a migration in the off chance they decide to use this for a long string.